packages/typespec-vscode/package.json

```diff
@@ -98,6 +98,15 @@
               "additionalProperties": false
             }
           },
+          "typespec.emitRequiredForCompile": {
+            "type": "array",
+            "default": [],
+            "description": "List of emitter names that should be run during compilation.",
```

Comment:
List of emitters that should be included in compilation in typespec LSP server when supporting LSP related functionalities.